package.json

```diff
@@ -4,62 +4,65 @@
   "description": "ATF Report generation",
   "main": "handler.js",
   "scripts": {
-    "start": "npm run kill:offline && BRANCH=local BUCKET=local AWS_XRAY_CONTEXT_MISSING=LOG_ERROR npm run start:offline && node_modules/serverless/bin/serverless invoke local -f main --path tests/resources/queue-event.json",
+    "start": "npm run kill:offline && BRANCH=local BUCKET=local SECRET_NAME=feature/atf-report/api-key AWS_XRAY_CONTEXT_MISSING=LOG_ERROR npm run start:offline && node_modules/serverless/bin/serverless invoke local -f main --path tests/resources/queue-event.json",
     "start:offline": "node_modules/gulp/bin/gulp.js start-serverless",
     "kill:offline": "node_modules/gulp/bin/gulp.js kill-serverless",
     "start:docker": "docker-compose -f docker-compose.yml up -d",
     "build": "node_modules/typescript/bin/tsc --rootDir ./ --outDir .build --sourceMap false && npm run build:copy",
     "build:dev": "node_modules/typescript/bin/tsc && npm run build:copy",
     "build:copy": "find src -type f \\( -name \"*.yml\" -or -name \"*.json\" -or -name \"*.xlsx\" \\) | cpio -pdm .build && find tests -type f \\( -name \"*.yml\" -or -name \"*.json\" \\)  | cpio -pdm .build",
     "build:docker": "docker-compose -f docker-compose.yml up -d --build",
-    "test": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.unitTest.ts\" --coverage",
+    "test": "AWS_XRAY_CONTEXT_MISSING=LOG_ERROR BRANCH=local SLS_DEBUG=* SECRET_NAME=feature/atf-report/api-key jest --testMatch=\"**/*.unitTest.ts\" --coverage",
     "test-i": "BRANCH=local jest --testMatch=\"**/*.intTest.ts\" --runInBand --passWithNoTests",
     "tslint": "tslint src/**/*.ts tests/**/*.ts",
     "prepush": "npm test && npm run tslint",
     "security-checks": "git secrets --scan && git log -p | scanrepo",
-    "sonar-scanner": "BRANCH=local npm run tslint && npm run coverage && sonar-scanner",
+    "sonar-scanner": "BRANCH=local export SECRET_NAME=feature/atf-report/api-key && npm run tslint && npm run coverage && sonar-scanner",
     "coverage": "jest --testMatch=\"**/*.*Test.ts\" --coverage",
     "audit-dep": "npm audit"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "aws-lambda": "^0.1.2",
-    "aws-sdk": "^2.560.0",
-    "aws-xray-sdk": "^2.4.0",
-    "exceljs": "1.7.0",
+    "aws-sdk": "^2.581.0",
+    "aws-xray-sdk": "^2.5.0",
+    "exceljs": "^1.8.0",
+    "js-yaml": "^3.13.1",
     "moment": "^2.24.0",
     "moment-timezone": "^0.5.27",
-    "node-yaml": "^3.2.0",
-    "notifications-node-client": "^4.6.0",
+    "node-yaml": "^4.0.1",
```

Comment:
this will probably break things. node-yaml is .... just so evil.

Reply:
Tests ran fine